src/ApplicationCore/Extensions/GuardExtensions.cs

```diff
@@ -1,9 +1,10 @@
 ﻿using System.Collections.Generic;
 using System.Linq;
+using Ardalis.GuardClauses;
 using Microsoft.eShopWeb.ApplicationCore.Entities.BasketAggregate;
 using Microsoft.eShopWeb.ApplicationCore.Exceptions;
 
-namespace Ardalis.GuardClauses;
+namespace Microsoft.eShopWeb.ApplicationCore.Extensions;
```

Comment:
By using the same namespace as the other guards in the package, developers get intellisense/autocomplete with all of the guard extensions by just including the Ardalis.Extensions namespace. This is why it's counterintuitively used here.